src/docfx/serve/LanguageServerBuilder.cs

```diff
@@ -60,14 +61,17 @@ public async void Run(CancellationToken cancellationToken = default)
                     await WaitToTriggerBuild(cancellationToken);
 
                     using var progressReporter = await CreateProgressReporter();
+                    await Task.Yield();
```

Comment:
>await Task.Yield(); [](start = 20, length = 19)

what is this for?

Reply:
Add a description.

Reply:
Consider move this line right before calling `_builder.Build`, in case some other code that calls lsp API are inserted between lin 69 and line 73